crates/pixi_cli/src/workspace/channel/mod.rs

```diff
@@ -48,6 +48,26 @@ pub struct AddRemoveArgs {
     pub feature: Option<String>,
 }
 
+#[derive(strum::Display, strum::EnumString)]
+enum ReportOperation {
+    #[strum(to_string = "Added")]
+    Add,
+    #[strum(to_string = "Removed")]
+    Remove,
+    #[strum(to_string = "already exists")]
+    AlreadyExists,
+}
```

Comment:
I feel like, this would be clearer without `strum` and by just adding the text to `operation_text`.